src/Adapters/RestAdapter.ts

```diff
@@ -29,8 +29,12 @@ export class RestAdapter implements IAdapter<string> {
    *
    * @returns {string}
    */
-  public getRestURL() {
-    return `${this.apiUrl}/${this.pathName}`;
+  public getRestURL(id?: string) {
+    let url = `${this.apiUrl}/${this.pathName}`;
+    if (id) {
+      url += `/${id}`;
+    }
+    return url;
```

Comment:
This works. We could also do the following:
```
return [
  this.apiUrl,
  this.pathName,
  id
].join("/");
```
The `join` method will omit `id` when it's `null` or `undefined`.